server/game/cards/09.2-BoB/UtakuRumaru.js

```diff
@@ -0,0 +1,40 @@
+const DrawCard = require('../../drawcard.js');
+const { Players, CardTypes, Locations } = require('../../Constants');
+const AbilityDsl = require('../../abilitydsl');
+
+class UtakuRumaru extends DrawCard {
+    setupCardAbilities() {
+        this.persistentEffect({
+            match: (card) => card.isHonored && card.type === CardTypes.Character,
+            targetController: Players.Self,
+            effect: AbilityDsl.effects.modifyGlory(1)
+        });
+
+        this.persistentEffect({
+            match: (card) => card.isDishonored && card.type === CardTypes.Character,
+            targetController: Players.Self,
+            effect: AbilityDsl.effects.modifyGlory(-1)
+        });
+
+        this.reaction({
+            title: 'honor a participating character',
+            when: {
+                afterConflict: (event, context) => context.source.isParticipating() && event.conflict.winner === context.player
+            },
+            cost: AbilityDsl.costs.discardCard({
+                location: Locations.Hand,
+                targets: true
```

Comment:
As the card doesn't use the word 'choose' with respect to the discard this cost is not targeting.  Please remove the line `targets: true`.